new-pipeline/style/custom.css

```diff
@@ -0,0 +1,75 @@
+.button {
```

Comment:
Let's find existing bootstrap styles that we can use instead of adding custom CSS.
The text-transform in here is what makes the whole text upper-case.

Reply:
> The text-transform in here is what makes the whole text upper-case.

Thank you for pointing that out.

> Let's find existing bootstrap styles that we can use instead of adding custom CSS.

That would be difficult I guess, given the amount of css it is. However, if that's really needed, I shall work on it.

Reply:
@georgf Can I try and make it as least as possible maybe ?

Reply:
You don't need to go through all the CSS.
The project uses Bootstrap v2.0.3, the documentation for that can be found here:
https://getbootstrap.com/2.0.3/index.html